### PR TITLE
Add increase/decrease methods

### PR DESCRIPTION
**Pull request by ButeraMV**
LIST OF CHANGES:
-Added methods on the cart model
-Updated the update action on the controller
-
-

THINGS I'M UNSURE ON:
-
-
-

RSPEC GREEN?
YES

*MENTION GROUP MEMBER TO REVIEW PR*

### CODE REVIEW

**Comment by blsrofe** [comment]
Looks great!

File: app/models/cart.rb [hunk 1]
```diff
@@ -25,5 +25,23 @@ def cart_total
...
+
+  def quantity(item_id)
+    @contents[item_id.to_s]
+  end
+
+  def increase_item(item_id)
+    @contents[item_id.to_s] += 1
+  end
+
+  def decrease_item(item_id)
```
Comment (snayrouz) [inline app/models/cart.rb:37]: Great poro method for the update cart.

**Comment by sidewinder2020** [comment]
Simple, clean, easy to decipher methods - I like it, and it will be easy to work off of in the future

File: app/views/carts/show.html.erb [hunk 2]
```diff
@@ -5,7 +5,12 @@
...
+  <div class="col-md-2" id="item-<%= item.id %>-quantity">
+    Quantity: <%= @cart.quantity(item.id) %>
+    <%= link_to "+", cart_path(item_to_increase: item.id), method: :patch %>
+    <%= link_to "-", cart_path(item_to_decrease: item.id), method: :patch %>
+  </div>
   <%= link_to "Remove", cart_path(item), method: :delete %>
 <% end %>
 
-<p>Total Cart Price:<%= @cart.cart_total %></p>  
+<p>Total Cart Price:<%= number_to_currency(@cart.cart_total) %></p>
```
Comment (snayrouz) [inline app/views/carts/show.html.erb:16]: good erb

**Pull request merged**